partner_compassion/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+##############################################################################
```

Comment:
This should be merged in file `partner_compassion` as it extends the same class.